contracts/contracts/coordination/GlobalAllowList.sol

```diff
@@ -0,0 +1,55 @@
+pragma solidity ^0.8.0;
+import "@openzeppelin/contracts/access/AccessControlDefaultAdminRules.sol";
+import "@openzeppelin/contracts/utils/cryptography/ECDSA.sol";
+import "./IRitualAuthorizer.sol";
+import "./Coordinator.sol";
+
+contract GlobalAllowList is AccessControlDefaultAdminRules, IRitualAuthorizer {
+    using ECDSA for bytes32;
+
+    Coordinator public coordinator;
+    mapping(uint256 => mapping(address => bool)) public authorizations;
+
+    constructor(
+        Coordinator _coordinator,
+        address _admin
+    ) AccessControlDefaultAdminRules(0, _admin) {
+        coordinator = _coordinator;
+    }
+
+    function setCoordinator(Coordinator _coordinator) public {
+        require(hasRole(DEFAULT_ADMIN_ROLE, msg.sender), "Only admin can set coordinator");
+        coordinator = _coordinator;
+    }
+
+    function isAuthorized(
```

Comment:
Is the DKG client supposed to call this method at some point? Fore example, when picking nodes for the DKG cohort?